packages/headless/src/features/facets/range-facets/numeric-facet-set/numeric-facet-actions.ts

```diff
@@ -44,8 +45,10 @@ const numericFacetRegistrationOptionsDefinition = {
  */
 export const registerNumericFacet = createAction(
   'numericFacet/register',
-  (payload: NumericFacetRegistrationOptions) =>
-    validatePayload(payload, numericFacetRegistrationOptionsDefinition)
+  (payload: NumericFacetRegistrationOptions) => {
+    validateManualNumericValues(payload);
```

Comment:
Since we `validateManualNumericValues` before  bueno has had a chance to verify that the payload are actually numbers, there's a possibility that we would end up comparing values that are not numbers at all inside `validateManualNumericValues`.

So we could end up doing comparison like `if(potato > NaN)` inside `validateManualNumericValues`.

Which... might return kind of random message/unpredictable behaviour a bit.

Same comment goes for `registerDateFacet` action, which does the same thing.